packages/storybook/stories/components/tooltip/tooltip.stories.ts

```diff
@@ -1,31 +1,34 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
 import { ODS_TOOLTIP_POSITION, ODS_TOOLTIP_POSITIONS } from '@ovhcloud/ods-components';
 import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-tooltip';
-import { type Meta, StoryObj } from '@storybook/web-components';
-// import { html } from 'lit-html';
-// import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
-import { CONTROL_CATEGORY, orderControls } from '../../control.ts';
+import { html } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import { CONTROL_CATEGORY, orderControls } from '../../control';
 
 defineCustomElement();
 
 const meta: Meta = {
   title: 'ODS Components/User feedback/Tooltip',
   component: 'ods-tooltip',
-  // decorators: [(story) => html`<div style="height: 300px; display: inline-flex">${story()}</div>`],
+  parameters: {
+    layout: 'centered',
+  },
+  decorators: [(story) => html`<div style="padding-top: 50px; display: inline-flex;">${story()}</div>`],
```

Comment:
<img width="479" alt="Capture d’écran 2024-04-02 à 09 07 35" src="https://github.com/ovh/design-system/assets/1390934/7cacb44e-0ce7-482c-ab32-4d44c07ff3ab">

Annonying thing with decorator is that they appear on the canvas sources